SourceBitBucket/SourceBitBucket.php

```diff
@@ -304,7 +316,7 @@ public function import_commits( $p_repo, $p_commit_ids, $p_branch = '' ) {
 			echo "Retrieving $t_commit_id ... ";
 			$t_json = null;
 			if( empty($this->commits_cache[$t_commit_id]) ) {
-				$t_url  = $this->api_url20( "repositories/$t_username/$t_reponame/commit/$t_commit_id/" );
+				$t_url  = $this->api_url20( "repositories/$t_username/$t_reponame/diffstat/$t_commit_id" );
```

Comment:
This API call is successful in my test, yet import_commit() fails at line 330 because there is no `hash` key in the returned JSON. Is *diffstat* the correct endpoint to use ? It was *commit* before (already a 2.0 API call)

Reply:
commit ressource do not content files status, you need to use diffstat for get all changesets in files.
Example: https://api.bitbucket.org/2.0/repositories/bitbucket/geordi/diffstat/d222fa2